bigquery/cloud-client/query_params.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Command-line app to perform queries with parameters in BigQuery.
+
+For more information, see the README.md under /bigquery.
+
+Example invocation:
+    $ python query_params.py --use-named-params 'romeoandjuliet' 100
+    $ python query_params.py --use-positional-params 'romeoandjuliet' 100
+"""
+
+import argparse
+import datetime
+import time
+import uuid
+
+from google.cloud import bigquery
+import pytz
+
+
+def wait_for_job(job):
+    while True:
+        job.reload()  # Refreshes the state via a GET request.
+        if job.state == 'DONE':
+            if job.error_result:
+                raise RuntimeError(job.errors)
+            return
+        time.sleep(1)
+
+
+def print_results(query_results):
+    """Print the query results by requesting a page at a time."""
+    page_token = None
+
+    while True:
+        rows, total_rows, page_token = query_results.fetch_data(
+            max_results=10,
+            page_token=page_token)
+
+        for row in rows:
+            print(row)
+
+        if not page_token:
+            break
+
+
+def query_positional_params(corpus, min_word_count):
+    client = bigquery.Client()
+    query = """SELECT word, word_count
+    FROM `bigquery-public-data.samples.shakespeare`
+    WHERE corpus = ?
+    AND word_count >= ?
+    ORDER BY word_count DESC;
+    """
+    query_job = client.run_async_query(
+        str(uuid.uuid4()),
+        query,
+        query_parameters=(
+            bigquery.ScalarQueryParameter(
+                # Set the name to None to use positional parameters (? symbol
+                # in the query).  Note that you cannot mix named and positional
+                # parameters.
+                None,
+                'STRING',
```

Comment:
It doesn't seem necessary to break these into lines, you an just put `None, 'STRING', corpus` on the same line.